src/main/java/echoServer/ServerSocketWrapperInterface.java

```diff
@@ -0,0 +1,7 @@
+package echoserver;
+public interface ServerSocketWrapperInterface {
```

Comment:
let's be consistent with the spacing. I think there' usually a space between the package line and the next